src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -40,7 +40,7 @@
                          :aggregated-births BirthsSchema
                          :aggregated-deaths DeathsOutputSchema}}
   [{:keys [historic-population]} {:keys [first-proj-year]}]
-  {:aggregated-population (wds/select-from-ds historic-population {:year (dec first-proj-year)})
+  {:aggregated-population historic-population
```

Comment:
this needs to pull out all the historic data up to but not including the `first-proj-year` so very close to this `{:aggregated-population (wds/select-from-ds historic-population {:year (dec first-proj-year)})` but also including previous years